src/features/post/components/Category.tsx

```diff
@@ -0,0 +1,13 @@
+import {Badge} from "@/components/shadcn-ui/badge";
+import {FC} from "react";
+
+interface CategoryProps extends React.HTMLAttributes<HTMLElement> {
+  "data-testid": string;
+}
+
+export const Category: FC<CategoryProps> = ({
+  children,
```

Comment:
get the children (string) then pass to the to_hex or to_rgb function to get the corresponding color then 
use that color as badge bacckground